### DISCUSSION TITLE
关于RHEL下服务没法启动的疑问请教

### DISCUSSION THREAD

**Discussion by gisxlab** [jeessy2/ddns-go#784] [Q&A] [answered]
ddns-go版本：5.5.0
系统版本：rhel8.8

> 无论是默认安装，还是带参数安装，web端都没法访问/端口没被监听；
> 而CentOS则没有这个问题。

确定是系统原因造成的，但不知道是系统哪里的原因，请问有没有大佬折腾过？

**Comment by WaterLemons2k** [comment]
~开放端口了吗？~

[Custom systemd service ExecStart fails with AVC denials - Red Hat Customer Portal](https://access.redhat.com/solutions/5248291)

**Reply by WaterLemons2k** [reply]
有没有禁用 SELinux？

**Reply by gisxlab** [reply]
- 放行了，9876/9899/19876都试过，防火墙也关闭测试了
`[root@geohub ddnsgo]# firewall-cmd --zone=public --list-ports`
`80/tcp 443/tcp 9899/tcp 19876/tcp`

- SELinux是开启的，CentOS那边开启了是没问题的。

**Reply by WaterLemons2k** [reply, marked as answer]
> * SELinux是开启的，CentOS那边开启了是没问题的。

这是只有 RHEL 8 才会有的 bug，要么应用更新 [RHBA-2022:1995 - Bug Fix Advisory - Red Hat 客户门户网站](https://access.redhat.com/errata/RHBA-2022:1995)，要么禁用 SELinux。

**Reply by gisxlab** [reply]
谢谢大佬，确认是SELinux原因造成的！赞
![0061969A](https://github.com/jeessy2/ddns-go/assets/35869860/9780ccf6-b0af-436c-b15e-1de5ecebf1e8)
